crud/src/main/java/pironeer/crud/repository/domain/Comment.java

```diff
@@ -0,0 +1,46 @@
+package pironeer.crud.repository.domain;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long commentId;
+
+    @NotNull
+    private String content;
+
+    @NotNull
+    @Builder.Default
+    private LocalDateTime createdAt = LocalDateTime.now();
```

Comment:
```suggestion
    @NotNull
    @CreatedDate
    private LocalDateTime createdAt;
```

이 애노테이션을 사용해 보세요!
class에 `@EntityListeners(AuditingEntityListener.class)`, Application에 `@EnableJpaAuditing`도 추가하시면 돼요👍